src/js/apps/patients/view/view_app.js

```diff
@@ -54,4 +59,20 @@ export default App.extend({
 
     return groups;
   },
+  _filtersById(viewId, currentClinician) {
+    const clinician = currentClinician.id;
+
+    const filters = {
+      'owned-by-me': { clinician },
+      'actions-for-coordinators': { role: 'coordinator' },
+      'new-actions': { created: moment().subtract(1, 'day').format() },
```

Comment:
I think we should specifically do `subtract(24, 'hours')` here.
Then in the other spots we want `moment().startOf('day').subtract(n, 'days')`